lib/apparmortest.pm

```diff
@@ -632,6 +632,38 @@ sub adminer_database_delete {
     clear_console;
 }
 
+# Yast2 Apparmor set up
+sub yast2_apparmor_setup {
+    # Check Apparmor is active
+    systemctl("is-active apparmor");
+
+    # Turn to x11 and start "xterm"
+    select_console("x11");
+    x11_start_program("xterm");
```

Comment:
In the file, you used the **x11_start_program('xterm');** before. I think they are both okay.

Reply:
Yes, I used `'$str'` before, but I think `"$str"` is enough so I prefer  `"$str"` if there is no special characters in `$str`. Of course we must chose  `'$str'` if there special characters in it. e.g., `$ `